src/React.AspNet.Middleware/React.AspNet.Middleware.csproj

```diff
@@ -5,7 +5,7 @@
     <Copyright>Copyright 2014-Present Facebook, Inc</Copyright>
     <AssemblyTitle>ReactJS.NET - Babel middleware for ASP.NET Core</AssemblyTitle>
     <Authors>Daniel Lo Nigro</Authors>
-    <TargetFrameworks>net451;netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
add target 451 or 461 too

Reply:
I wrote it on the pull request description, but with netstandard2.0 you do not requiere a different target if you use the library in a application targeting .NET Framework 4.6.1, that's the whole point of .NET Standard.